### PR TITLE
Health Check | CopyLinkedAnalystNotes - fix TypeError 

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/33797
fixes: https://github.com/demisto/etc/issues/33796

## Description
Fixed an issue when sorting **main_incident_grid** failed due to TypeError. Setting a default value of an empty string solved the problem.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by abaumgarten** [comment]
Waiting to @altmannyarden to approve that my fix solved the problem locally.

**Comment by altmannyarden** [comment]
Approved :) tested - work as expected ! Thank you

**Review by guykeller — APPROVED** [review]

File: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_1_13.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### CopyLinkedAnalystNotes
+- Fixed an issue when sorting **main_incident_grid** failed due to TypeError.
```
Comment (amshamah419) [inline Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_1_13.md:4]:
```suggestion
- Fixed an issue where sorting **main_incident_grid** failed due to TypeError.
```

File: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_1_13.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
+
+##### GetFailedTasks
+- Fixed an issue where empty failed incidents list created a null value in the context.
```
Comment (amshamah419) [inline Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_1_13.md:8]:
```suggestion
- Fixed an issue where an empty failed incidents list created a null value in the context.
```

**Pull request merged**